src/routes/(blog-article)/how-to-setup-the-development-environment/+page.md

```diff
@@ -256,17 +265,28 @@ For more details about the Torrust Index Backend, check the [Index Backend docum
 
 At the time of writing, the Frontend requires:
 
-- Node: `^19.0.0`
+- Node: `^21.7.3`
 
 The frontend is a [Nuxt](https://nuxt.com/) application.
 
+The last repository we need to upload is the torrust-index-gui repository . Remember to install Node.js and we need to be in the torrust folder to clone the repository.
+
+<CodeBlock lang="terminal">
+
+```bash
+  git clone https://github.com/torrust/torrust-index-gui.git
+
+  npm install
+```
+
+</CodeBlock>
```

Comment:
The las repository we need to **download** ...

And you can change the command to:

```
git clone https://github.com/torrust/torrust-index-gui.git
cd torrust-index-gui.git
npm install
npm run dev
```

TO use one single command.